toolchain/semantics/semantics_builtin_kind.def

```diff
@@ -53,6 +53,10 @@ CARBON_SEMANTICS_BUILTIN_KIND(IntegerType, TypeType)
 // we may not want it this way, but for now this is the approach.
 CARBON_SEMANTICS_BUILTIN_KIND(RealType, TypeType)
 
+// The canonical empty tuple. Long-term we may want to change the
+// representation, but for now it's an easy way to identify uses.
+CARBON_SEMANTICS_BUILTIN_KIND(EmptyTuple, TypeType)
```

Comment:
Seems fine for now. Long-term, the type of an empty tuple value should be an empty tuple type, whose type in turn is `type`.